### ISSUE TITLE
last_vacuum / last_analyze - Add exclude filter for table and schema

### ISSUE THREAD

**Issue by robinportigliatti** [OPMDG/check_pgactivity#369]
Hello,

I have a customer whose always creating and destroying working tables on a specific schema or a specific table.

Would it be possible to add two parameters to excludes tables and / or schema ?

Robin,

**Comment by yhuelf** [comment]
Indeed, this probe could support an `--exclude REGEX` parameter with the same semantics as for btree_bloat.

**Comment by gleu** [comment]
It will be on the next release, on the 18th of may. I'm closing the issue.
